src/java.xml/share/legal/xmlxsd.md

```diff
@@ -26,7 +26,7 @@ modifications:
     [$year-of-document] World Wide Web Consortium.
     https://www.w3.org/copyright/software-license-2023/"
 
-Disclaimers §anchor
```

Comment:
This is an incorrectly copied piece of html; compare how the very same license is handled in e.g. `src/java.xml/share/legal/schema10part1.md`. The § is the non-ascii character that triggered my detection of this, but the entire "anchor" string is incorrect here.